src/probe.cpp

```diff
@@ -36,29 +36,33 @@ Probe Probe::from_csv(const std::string &line) {
         probe.protocol = Protocols::l4_from_string(token);
         break;
       case 5:
+        probe.flow_label = Checked::stou32(token);
+        break;
+      case 6:
         probe.wait_us = Checked::stou32(token);
         break;
+
       default:
         break;
     }
     index++;
   }
-  if (index < 5 || index > 6) {
+  if (index < 6 || index > 7) {
```

Comment:
Can we make this `index < 5 || index > 7` so that specifying the flow label is not mandatory?

Reply:
How do we know then which optional field to select between the flow_label and the wait_us?

Reply:
If the user wants to set `wait_us` he can just set `flow_label` to 0.

Reply:
So `flow_label` is always 6th and `wait_us` 7th.

Reply:
OK, sounds good